### PR TITLE
Use the in memory store on the Nano server

### PR DESCRIPTION
**Pull request by victorhurdugaci**
The SQL client doesn't work well on the Nano server so for now we'll use the in memory store.

Please review @suhasj

### CODE REVIEW

File: src/MusicStore/Startup.cs [hunk 1]
```diff
@@ -36,8 +36,9 @@ public void ConfigureServices(IServiceCollection services)
         {
             services.Configure<AppSettings>(Configuration.GetSubKey("AppSettings"));
 
-            //Sql client not available on mono
-            var useInMemoryStore = _runtimeEnvironment.RuntimeType.Equals("Mono", StringComparison.OrdinalIgnoreCase);
+            // Use the in memory store when running on mono or on the Nano server
+            var useInMemoryStore = _runtimeEnvironment.RuntimeType.Equals("Mono", StringComparison.OrdinalIgnoreCase) ||
```
Comment (suhasj) [inline src/MusicStore/Startup.cs]: Can you move the check condition for Mono in the Platform class ? You can replace this usage through out the project making it consistent

**Comment by victorhurdugaci** [comment]
@suhasj updated

**Comment by suhasj** [comment]
Can you make the changes in this file too https://github.com/aspnet/MusicStore/blob/dev/src/MusicStore/StartupOpenIdConnect.cs

**Comment by suhasj** [comment]
:shipit:

**Pull request merged**